Modules/_ctypes/_ctypes.c

```diff
@@ -2609,6 +2698,10 @@ PyCData_traverse(CDataObject *self, visitproc visit, void *arg)
 {
     Py_VISIT(self->b_objects);
     Py_VISIT((PyObject *)self->b_base);
+    PyTypeObject *type = Py_TYPE(self);
+    if (type->tp_flags & Py_TPFLAGS_HEAPTYPE) {
+        Py_VISIT(type);
+    }
```

Comment:
```suggestion
    Py_VISIT(type);
```
There seems to be no static type left.